content/documentation/edge/installation-guide/index.adoc

```diff
@@ -83,7 +83,7 @@ If you are installing on OpenShift, you must grant the `cluster-admin` role to t
 
 == Install Kiali via Istio or Maistra
 
-icon:lightbulb[size=1x]{nbsp} Both Istio and Maistra include Kiali distributions.
+icon:bullhorn[size=1x]{nbsp} Starting with Istio 1.7, Kiali is no longer shipped with the istioctl installer. If you are using an  Istio implementation of version 1.7 or higher, this section is not valid for you.
```

Comment:
You should tell them what IS valid, maybe link them there.

Reply:
Changing to: `If you are using an  Istio implementation of version 1.7 or higher, skip this section and go on to the next.`